### ISSUE TITLE
GA are compatibility with NWChem?

### ISSUE THREAD

**Issue by cnelson3** [nwchemgit/nwchem#36]
Hi All,
I've a user who's reported an issue similar to those reported by Eduoardo in https://github.com/GlobalArrays/ga/issues/95

Before I set about rebuilding NWChem to fix this, is it advisable to use GA-v5.6.4, or should I use a later version of GA? e.g. v5.6.5, or v5.7?

thanks,
Chris.

**Comment by jeffhammond** [comment]
Just use https://github.com/nwchemgit/nwchem/blob/master/src/tools/get-tools-github like this:
```sh
cd $NWCHEM_TOP/src/tools
./get-tools-github
```

I guess there have been compatibility issues in the past, but I don't remember any in a really long time.  I think every version of GA 5 should work with recent NWChem, up to simple bugs.

**Comment by jeffhammond** [comment]
You might also clone NWChem and GA from Github on a machine that is online, and then `rsync` the whole thing over to your offline machine.  A more advanced way to do it would be to set the remote repo to a local machine that can access both Github and your internal machine.  You can then bounce the repo through it, where the local machine serves as the git remote for the system without internet access.

**Comment by jeffhammond** [comment]
Actually, based upon this, if you put `ga-5.6.5.tar.gz` in `$NWCHEM_TOP/src/tools`, it shouldn't need to download it.
```
if [ ! -f ga-"$GA_RELEASE_NO".tar.gz ]; then
        TAR_URL=https://github.com/GlobalArrays/ga/releases/download/v"$GA_RELEASE_NO"/ga-"$GA_RELEASE_NO".tar.gz
        # TAR_URL=https://github.com/edoapra/ga/releases/download/v"$GA_RELEASE_NO"/ga-"$GA_RELEASE_NO".tar.gz
        #         https://github.com/edoapra/ga/releases/download/v5.6.5/ga-5.6.5.tar.gz
        echo "downloading" ga-"$GA_RELEASE_NO".tar.gz  "from" "$TAR_URL"
        WGET_YES=`wget  -O 2>&1 | head -1  | awk ' /require/ {print "Y";exit};{print "N"}'`
        if [ $WGET_YES = "Y" ]
        then
            wget  -O ga-"$GA_RELEASE_NO".tar.gz "$TAR_URL"
        else
            #try curl because wget is not there
            curl -L  "$TAR_URL" -o ga-"$GA_RELEASE_NO".tar.gz
        fi
```

**Comment by edoapra** [comment]
@cnelson3 The version of GA I would suggest you to use is 5.6.5
As far as NWChem is concerned, I would advise you to use the hotfix/release-6-8 branch, that contains all the 6.8 code plus bug fixes after the 6.8 release.
To checkout
`git clone -b hotfix/release-6-8 http://github.com/nwchemgit/nwchem nwchem-6.8.1`
Cheers, Edo

**Comment by cnelson3** [comment]
Thanks both for the advice - I'll give it a try.
